react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+let x = 0;
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { onClicked: x };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    x++;
+    this.setState({ onClicked: x });
+  }
```

Comment:
you should not create a variable outside of the class for state. this limits you to only using the component once. if you put two custom buttons on the page, you'll see what i mean

```jsx
root.render(
  <div>
    <HotButton />
    <HotButton />
  </div>
)
```